From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tim Schumacher <timschumi@gmx.de>
Date: Fri, 1 Sep 2023 23:07:15 +0200
Subject: [PATCH] Ignore unknown warning options

`-Wlogical-op`, `-Wmissing-parameter-type`, and `-Wold-style-declaration`
are unknown to Clang. While this isn't fatal by itself, it is very
noisy.
---
 generate/unix/Makefile.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/generate/unix/Makefile.config b/generate/unix/Makefile.config
index 0deeb14115c1473ef0e1b55f6e174122ae52633d..e255cddf11ba4801daabe835e7aa2a9840825ca0 100644
--- a/generate/unix/Makefile.config
+++ b/generate/unix/Makefile.config
@@ -243,6 +243,7 @@ ifneq ($(ACPI_HOST), _FreeBSD)
             -Wlogical-op\
             -Wmissing-parameter-type\
             -Wold-style-declaration\
+            -Wno-unknown-warning-option\
             -Wtype-limits
     endif
 endif
